src/issues/currentIssue.ts

```diff
@@ -130,19 +129,30 @@ export class CurrentIssue extends Disposable {
 		return undefined;
 	}
 
```

Comment:
Checking that the branch name starts with `origin/` isn't enough, as not all remotes will be named `origin`.

```suggestion

	private getRemoteBranchPrefix(branch: string): string | undefined {
		const remote = this.manager.repository.state.remotes.find((remote) => {
			return branch.startsWith(`${remote.name}/`);
		});
		return remote ? `${remote.name}/` : undefined;
	}

```